src/main/java/org/pac4j/vertx/handler/impl/Pac4jAuthHandlerOptions.java

```diff
@@ -26,6 +26,7 @@
     private String clientName = "";
```

Comment:
I'm happy to use plurals, but I still think the variables should reflect that they are names rather than actual clients.